client/lib/checkout/payment-methods.tsx

```diff
@@ -8,18 +8,21 @@ import creditCardPlaceholderImage from 'calypso/assets/images/upgrades/cc-placeh
 import creditCardUnionPayImage from 'calypso/assets/images/upgrades/cc-unionpay.svg';
 import creditCardVisaImage from 'calypso/assets/images/upgrades/cc-visa.svg';
 import payPalImage from 'calypso/assets/images/upgrades/paypal.svg';
+import razorpayImage from 'calypso/assets/images/upgrades/upi.svg';
```

Comment:
I'm assigning the UPI logo to `razorpayImage` here since we're using it for the logo instead of the actual Razorpay logo.